modules/fluid_properties/src/fluidproperties/NaClFluidProperties.C

```diff
@@ -137,19 +137,42 @@ NaClFluidProperties::cp_from_p_T(Real pressure, Real temperature) const
 {
   // Correlation needs pressure in bar
   Real pbar = pressure * 10.0e-5;
-  // Correlation requires temperature in Celcius
+  // Correlation requires temperature in Celsius
   Real Tc = temperature - _T_c2k;
   // Triple point temperature of NaCl (in C)
   Real Tt = _T_triple - _T_c2k;
   // Coefficients used in the correlation
   Real r3 = -1.7099e-3 - 3.82734e-6 * Tc - 8.65455e-9 * Tc * Tc;
   Real r4 = 5.29063e-8 - 9.63084e-11 * Tc + 6.50745e-13 * Tc * Tc;
 
-  // Halite isobaric heat capapcity
+  // Halite isobaric heat capacity
   return 1148.81 + 0.551548 * (Tc - Tt) + 2.64309e-4 * (Tc - Tt) * (Tc - Tt) + r3 * pbar +
          r4 * pbar * pbar;
 }
 
+void
+NaClFluidProperties::cp_from_p_T(
+    Real pressure, Real temperature, Real & cp, Real & dcp_dp, Real & dcp_dT) const
+{
+  // Correlation needs pressure in bar
+  Real pbar = pressure * 10.0e-5;
```

Comment:
Conversion to bar should be 1e-5 (1 bar = 1e5 Pa)

Reply:
very true. this is like this all over this property :O

Reply:
fixed